facebookads/api.py

```diff
@@ -123,7 +123,8 @@ def is_transient(self):
 
         json_body = self.json()
         try:
-            return json_body.get('error', {}).get('is_transient', False)
+            error = json_body.get('error', {})
+            return error.get('is_transient', False) or error.get('message', False) == 'An unknown error occurred'
```

Comment:
you don't check for the HTTP status?

Reply:
@m-vdb done just above with the is_success method.

Reply:
✅